charts/promitor-scraper/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "promitor-scraper.fullname" . }}
+  labels:
+    app: {{ template "promitor-scraper.name" . }}
+    chart: {{ template "promitor-scraper.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: {{ template "promitor-scraper.name" . }}
+      release: {{ .Release.Name }}
+      type: {{ .Values.service.selectorType }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "promitor-scraper.name" . }}
+        release: {{ .Release.Name }}
+        type: {{ .Values.service.selectorType }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
Does this need a check if using secrets?

Reply:
Fixed